### PR TITLE
Added responsive fonts

### PR DESCRIPTION
**Pull request by Zxun2**
Changed the unit of measurement to rem (for fonts), and added some mixins for responsiveness. Let me know if anything else needs to be changed :)

Issue here: [https://github.com/NUS-Fintech-Society/SD_Fintech_Website/issues/57](url)

### CODE REVIEW

File: styles/variables.scss [hunk 1]
```diff
@@ -20,13 +20,13 @@ $font-weight-regular: 400;
...
-$font-size-1: 13px;
-$font-size-2: 15px;
-$font-size-3: 20px;
-$font-size-4: 24px;
-$font-size-5: 30px;
-$font-size-6: 35px;
-$font-size-7: 40px;
-$font-size-8: 45px;
-$font-size-9: 50px;
+$font-size-0: 1.8rem;
```
Comment (jerrenss) [inline styles/variables.scss:23]: I see you added 8px to each of the originals, now it looks slightly big haha. Shall we follow this guideline instead? Feel free to change it up a little if you think the sizes are strange.

image

Source: https://w3-lab.com/website-font-size-guidelines/

**Review by jerrenss — CHANGES_REQUESTED** [review]
Nice work! Just that the Event page's fonts look a little off hmm, you might wanna check out why. The regular body texts are too large and the headers are too small.

image

**Comment by Zxun2** [comment]
> Nice work! Just that the Event page's fonts look a little off hmm, you might wanna check out why. The regular body texts are too large and the headers are too small.
> 
> image

Fixed the weird sizing on the events page. Hopefully, things work out now! 

Issue here: [https://github.com/NUS-Fintech-Society/SD_Fintech_Website/issues/57](url)

**Comment by jerrenss** [comment]
@Zxun2 could you help to resolve the merge conflicts? Update your master branch locally, and merge it into your working branch. Then manually resolve the conflicts ya

**Comment by Zxun2** [comment]
> @Zxun2 could you help to resolve the merge conflicts? Update your master branch locally, and merge it into your working branch. Then manually resolve the conflicts ya

Done! 

Issue here: [https://github.com/NUS-Fintech-Society/SD_Fintech_Website/issues/57](url)

**Comment by jerrenss** [comment]
Hey @Zxun2 sflr, didn't see the notification. Could you help to check why there's this gap in the footer? Zhikai made a fix so there shouldn't be the contact form anymore in the production version

image

**Comment by Zxun2** [comment]
> Hey @Zxun2 sflr, didn't see the notification. Could you help to check why there's this gap in the footer? Zhikai made a fix so there shouldn't be the contact form anymore in the production version
> 
> image
> 
> Also, could you can make the sub-draweritems smaller than the draweritems? image
> 
> Other than that, looks good :)

Done! Forgot to add changes previously haha. Also, did the tweak to the font size on the side drawer.   

Issue here: https://github.com/NUS-Fintech-Society/SD_Fintech_Website/issues/57

**Review by jerrenss — APPROVED** [review]

**Pull request merged**